### ISSUE TITLE
False negative for W1201

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#182]
Originally reported by: **the mulhern (BitBucket: [the_mulhern](http://bitbucket.org/the_mulhern))**

---

For this code:

```
#!python

import logging

def do_a_thing():
    log = logging.getLogger("domain")
    log.debug("%s" % "junk")
```

a W1201 error is reported.

For this code, which is nearly identical:

```
#!python

import logging

def do_a_thing():
    logging.getLogger("domain").debug("%s" % "junk")
```

no such error is reported.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/182
